### PR TITLE
Adding Vectorize.jl (tag v0.0.3)

### PR DESCRIPTION
**Pull request by rprechelt**
This pull request contains `Vectorize.jl`, the cross-platform vectorization library developed for Julia's GSOC program. Contains tags for `v0.0.1` through `v0.0.3`. 

@ArchRobison @simonbyrne @vtjnash

### CODE REVIEW

**Comment by rprechelt** [comment]
Strange git error here: `ERROR: LoadError: Invalid url scheme http for package Vectorize. Should be 'git' or 'https'`. 

I'll see if I can't look into why this is happening.

File: Vectorize/url [hunk 1]
```diff
@@ -0,0 +1 @@
+http://github.com/rprechelt/Vectorize.jl
```
Comment (tkelman) [inline Vectorize/url]: this should be https

File: Vectorize/versions/0.0.3/requires [hunk 2]
```diff
@@ -0,0 +1 @@
+julia 0.5
```
Comment (tkelman) [inline Vectorize/versions/0.0.3/requires]: this will not be installable yet

Comment (rprechelt) [inline Vectorize/versions/0.0.3/requires]: Good point. Maybe this ought to wait until `0.5` is released.

Comment (tkelman) [inline Vectorize/versions/0.0.3/requires]: or you could do `0.5.0-rc0` or similar and it'll work for RC's or newer

**Pull request closed without merging**